### PR TITLE
Implement RetirementReport format

### PR DESCRIPTION
**Pull request by samsondav**
core ref: 5edbb5a9d33514017e496044e7b3e4962eb55dfa

### CODE REVIEW

**Review by brunotm — APPROVED** [review]

**Review by cedric-cordenier — DISMISSED** [review]

File: pkg/types/provider_llo.go [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+	ocrtypes "github.com/smartcontractkit/libocr/offchainreporting2plus/types"
+
 	"github.com/smartcontractkit/chainlink-common/pkg/types/llo"
 )
 
+type LLOConfigProvider interface {
+	OffchainConfigDigester() ocrtypes.OffchainConfigDigester
+	// One instance will be run per config tracker
+	ContractConfigTrackers() []ocrtypes.ContractConfigTracker
+}
```
Comment (jmank88) [inline pkg/types/provider_llo.go:13]: Don't we need GRPC/protos for this?

Comment (jmank88) [reply pkg/types/provider_llo.go:13]: Oh, I didn't realize these are only superficially connected to the Relayer interfaces :thinking:

Comment (samsondav) [reply pkg/types/provider_llo.go:13]: Do we? I don't know how that would work. We always returned a contract config tracker here

Comment (jmank88) [reply pkg/types/provider_llo.go:13]: If we had support for the rest, then this wouldn't compile until you filled in the new ones, but we just have it stubbed out to error I guess :shrug:

**Review by jmank88 — APPROVED** [review]

**Review by martin-cll — APPROVED** [review]

**Review by brunotm — APPROVED** [review]

**Pull request merged**